packages/http-console-formatter/src/templates/console/request.ts

```diff
@@ -3,16 +3,16 @@ import { OutgoingHttpHeaders } from 'http';
 import { formatHeadersMap, formatLength } from '../utils';
 
 export function request(
-  method: string,
-  uri: string,
-  headers: OutgoingHttpHeaders,
+  method?: string,
+  uri?: string,
+  headers?: OutgoingHttpHeaders,
```

Comment:
proč je tohle optional? ta HTTP log entry by zrovna tyhle property měla mít definovaný vždycky a odjinud se tohle nepoužívá, so..

Reply:
protoze to prijde z entry.data, kde je to optional